modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -220,6 +220,10 @@ style the component:
 
 **There should be no developer impact.**
 
+### Select in Preview
+
+Removed the `Select` component that was in `@workday/canvas-kit-preview-react` and replaced with the `Select` component in `@workday/canvas-kit-react`. Please see the `Select` documentation [here](https://workday.github.io/canvas-kit/?path=/docs/components-inputs-select--basic).
```

Comment:
This is our fault, but this should live in `12.0-UPGRADE-GUIDE.mdx` file since this is going into `major`